index.html

```diff
@@ -9,7 +9,9 @@
 	</head>
 	<body>
 		<div class="container-detail">
-			<img class="image-container" />
+			<div class="image-container">
+				<img class="image-cover" />
```

Comment:
is it necessary to add another tag? try to make the same using only the image-cover class